pkgs/by-name/ff/ff2mpv-rust/package.nix

```diff
@@ -3,35 +3,61 @@
 , fetchFromGitHub
 }:
 
+let
+  firefoxPaths = [
+    "lib/mozilla/native-messaging-hosts"
+
+    # wrapFirefox only links lib/mozilla path, so this is ineffective
+    # Still the above path works, despite documentation stating otherwise
+    # See: https://librewolf.net/docs/faq/#how-do-i-get-native-messaging-to-work
+    # "lib/librewolf/native-messaging-hosts"
+  ];
+
+  chromiumPaths = [
+    "etc/chromium/native-messaging-hosts"
+    "etc/opt/vivaldi/native-messaging-hosts"
+    "etc/opt/chrome/native-messaging-hosts"
+    "etc/opt/edge/native-messaging-hosts"
+  ];
+in
+
 rustPlatform.buildRustPackage rec {
   pname = "ff2mpv-rust";
-  version = "1.1.1";
+  version = "1.1.3";
 
   src = fetchFromGitHub {
     owner = "ryze312";
     repo = "ff2mpv-rust";
     rev = version;
-    hash = "sha256-sofv5uRLNbMT+w+ZDGjtKqBjYJk+UDzUDQrOiWvl5Hs=";
+    hash = "sha256-gk3MqoMCau6RTlttTLqFtlXhrYMOf89YFg8aAlh++9s=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
```

Comment:
@ryze312 Is this still needed? I thought you have the lockfile in your repo now.

Reply:
Is the hash optional when providing hash through over means, in this case  through lock file? If so, this can be removed.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md

Reply:
Looking at this, it seems the preferred way is to provide the `cargoHash` instead of `cargoLock` and then leave out `hash` from `src`. Did I get that correctly?

Reply:
The `src` should be left as it is now, you only have to provide a `cargoHash` instead of `cargoLock`.

Reply:
Ah I see, thought @eNV25 was pointing out that `hash` in `src`.